.azure-pipelines/linux-conda-CI-tf-keras.yml

```diff
@@ -45,6 +45,7 @@ jobs:
       python -m pip install --upgrade pip
       conda config --set always_yes yes --set changeps1 no
       pip install $(ONNX_PATH)
+      pip install h5py==2.9.0
```

Comment:
why does CI build needs this file too?

Reply:
keras-applications depends on h5py. Today it released 2.10.0 and build wheel fails. so specify 2.9.0.